modules/reporting/app/models/cost_query/result.rb

```diff
@@ -68,6 +68,10 @@ def end_timestamp
       start + self["units"].to_f.hours
     end
 
+    def entity_gid
+      "gid://#{GlobalID.app}/#{self['entity_type']}/#{self['entity_id']}"
```

Comment:
GlobalID provides a low-level method to construct `URI::GID`:

```suggestion
      URI::GID.build([GlobalID.app, *self.values_at('entity_type', 'entity_id'), nil])
```

Reply:
I'll use the `URI::Gid.create` method: https://community.openproject.org/work_packages/64975/activity

Reply:
Personally, I like the string construction better here